File: SimpleChaos/demo-applications/spring-demo/src/main/java/com/example/demo/Application.java
package com.example.demo;

import com.languagelatte.simplechaos.spring.SimpleChaosSpringConfig;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.annotation.Import;


@SpringBootApplication
@Import(SimpleChaosSpringConfig.class)
public class Application {

	public static void main(String[] args) {
		SpringApplication.run(Application.class, args);
	}
}

File: SimpleChaos/demo-applications/kotlin-spring-demo/src/main/kotlin/com/example/demo/Application.kt
package com.example.demo

import org.springframework.boot.autoconfigure.SpringBootApplication
import com.languagelatte.simplechaos.spring.SimpleChaosSpringConfig
import kotlin.jvm.JvmStatic
import org.springframework.boot.SpringApplication
import org.springframework.context.annotation.Import

@SpringBootApplication
@Import(SimpleChaosSpringConfig::class)
open class Application

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}